service/chartconfig/v1/resource/chart/current.go

```diff
@@ -15,12 +16,18 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	}
 
 	releaseContent, err := r.helmClient.GetReleaseContent(customObject)
-	if err != nil {
+	if helm.IsReleaseNotFound(err) {
+		// Fall through.
+		return &ChartState{}, nil
```

Comment:
This is not falling through. This is returning early. And why not `nil`?

Reply:
Ack, good point it is returning early. I updated the comment and it now returns nil.